scripts/build/build/targets.py

```diff
@@ -256,6 +252,9 @@ def HostTargets():
         app_targets.append(target.Extend(
             'ota-requestor', app=HostApp.OTA_REQUESTOR, enable_ble=False))
         app_targets.append(target.Extend('python-bindings', app=HostApp.PYTHON_BINDINGS))
+        app_targets.append(target.Extend('tv-app', app=HostApp.TV_APP))
+        app_targets.append(target.Extend('tv-casting', app=HostApp.TV_CASTING))
```

Comment:
name changed from tv-casting-app to tv-casting. this requires CI script updates (or do not rename)

Reply:
Did not realize this was a rename in my previous comment - title of the issue said "add" not move.

Reply:
You know what it is, you told me earlier to drop the -app since I was adding it. However, someone else added tv-casting-app as a non-crossbuilt target before this was able to be merged. I will modify to include the -app.

Reply:
OK updated, but for some reason the arm64 builds won't complete complaining missing avahi-client. I'll ping you on slack about this.